src/ui/layout/tests/Layout.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+
+import { Layout } from "..";
+
+
+describe('<Layout>', () => {
+    it("shows children", () => {
+        const child = <div data-testid="child">Test</div>;
```

Comment:
Don't user data-testid without a real reason. Replace this with fe. <input placeholder="SomeTitle" /> and use screen.getByPlaceholderText to search.

Also remove child variable - there is one good rule for creating variables. 
Create only when:
- reduces duplication,
- makes code cleaner

Reply:
done